packages/blade/src/components/Input/BaseInput/BaseInputVisuals.tsx

```diff
@@ -329,7 +329,13 @@ export const BaseInputVisuals = ({
             alignItems="stretch"
             alignSelf="stretch"
           >
-            {trailingInteractionElement}
+            <BaseBox
+              onClick={(e) => {
+                e.stopPropagation();
```

Comment:
With this if we click the eye icon in password input, I'm guessing the focus won't come to input 🤔 

Can we add `onTrailingInteractionElementClick` and add click handler to overall container including that padding space?

Then we can remove the click handler on icon and use that prop itself for handling clicks on icon